ui/src/utils/utils.js

```diff
@@ -356,11 +356,13 @@ export const htmlHintRules = [
 	{ rule: "attr-unsafe-chars", displayName: "Attributes - attributes cannot contain unsafe characters", ruleLink: "https://htmlhint.com/docs/user-guide/rules/attr-unsafe-chars" },
 	{ rule: "head-script-disabled", displayName: "HTML - the script tag cannot be used in another tag", ruleLink: "https://htmlhint.com/docs/user-guide/rules/head-script-disabled" },
 	{ rule: "code-block-missing-language", displayName: "Code block - missing language", ruleLink: "https://www.ssw.com.au/rules/set-language-on-code-blocks" },
-  	{ rule: "grammar-scrum-terms", displayName: "Grammar mistake - common Scrum terms", ruleLink: "https://www.ssw.com.au/rules/scrum-should-be-capitalized" },
+	{ rule: "grammar-scrum-terms", displayName: "Grammar mistake - common Scrum terms", ruleLink: "https://www.ssw.com.au/rules/scrum-should-be-capitalized" },
+	{ rule: "figure-format", displayName: "Figure Format - Check if Figure statement is wrapped in correct tag", ruleLink: "" },
  ];
 
  export const customHtmlHintRules = [
 	{ rule: "code-block-missing-language", displayName: "Code block - missing language", ruleLink: "https://www.ssw.com.au/rules/set-language-on-code-blocks" },
-  	{ rule: "grammar-scrum-terms", displayName: "Grammar mistake - common Scrum terms", ruleLink: "https://www.ssw.com.au/rules/scrum-should-be-capitalized" },
+	{ rule: "grammar-scrum-terms", displayName: "Grammar mistake - common Scrum terms", ruleLink: "https://www.ssw.com.au/rules/scrum-should-be-capitalized" },
+	{ rule: "figure-format", displayName: "Figure Format - Check if Figure statement is wrapped in correct tag", ruleLink: "" },
```

Comment:
1. Do you need to add the ruleLink?
2. Should the capital "F" in "Check if Figure" be lowercase?

Reply:
> Do you need to add the ruleLink?

Done - Added ruleLink

> Should the capital "F" in "Check if Figure" be lowercase?

Done - Fixed "F" to lowercase